[CUBRIDMAN-19] update broker_access_log (Eng/Kor)

nays111 opened the pull request:
http://jira.cubrid.org/browse/CUBRIDMAN-19 

update broker_changer usage in en/control.rst , ko/control.rst
update broker_access_log  in en/control.rst , ko/control.rst

nays111 committed:
Update control.rst

nays111 committed:
Update python.rst

nays111 committed:
fix broker_changer usage

nays111 committed:
update broker_access_log in english

nays111 committed:
Typo correction

hgryoo commented:
May I know why this PR doesn't have a related JIRA issue? It may hard to track why some content is changed later.
and the Update python.rst commit (e208d57) seems not to be related to the title of this PR.

nays111 commented:
@hgryoo    
I just updated PR's title not to confuse.  Update python.rst  ([e208d57](https://github.com/CUBRID/cubrid-manual/pull/199/commits/e208d57ad2f2c146022dbb2c796360fb3e8178a7)) was the job for 'travis ci build' in github. I updated it because of build failure. Do I need to make another PR for python.rst ?

hgryoo commented:
@nays111 yes, I think updating python.rst should be a separate PR. and you may ignore the failure of Travis now.

nays111 committed:
Revert "Update python.rst"

This reverts commit e208d57ad2f2c146022dbb2c796360fb3e8178a7.

nays111 committed:
Revert "fix broker_changer usage"

This reverts commit b0a6482be9c57e51ea83b3640b8e3768e2f9e67b.

nays111 committed:
fix broker_changer usage in kor
